tests/Integration/Database/EloquentBelongsToManyTest.php

```diff
@@ -48,7 +48,7 @@ protected function setUp(): void
 
         Schema::create('posts_tags', function (Blueprint $table) {
             $table->integer('post_id');
-            $table->integer('tag_id');
+            $table->string('tag_id');
```

Comment:
In one test we're storing the `name` as a value so this needs to be a string. SQLite accepted this but it's in fact an incorrect column type for these tests.